R/utils.R

```diff
@@ -36,3 +36,20 @@ reduce <- function(.x, .f, ..., .init) {
 
   v
 }
+
+check_vline_size <- function(data) {
+  if (is.null(data$linewidth) && !is.null(data$vline_size)) {
```

Comment:
This should be `is.null(data$vline_width) && !is.null(data$vline_size)` and then we need a second check for `linewidth` and `size`, respectively.

Reply:
Added